pallets/runtime/tests/exec_macro/src/lib.rs

```diff
@@ -39,6 +41,9 @@ impl Parse for Exec {
     }
 }
 
+/// Convert a direct call to a module to d extrinsic execution.
+/// When `#[integration-test]` is not set, this becomes a noop.
```

Comment:
`.. d extrinsic ...` typo?

`.. becomes a noop` this sounds like the extrinsic wouldn't be called.  Maybe something like "leaves it as a direct call".